website/docs/reference/warehouse-profiles/bigquery-profile.md

```diff
@@ -195,7 +195,97 @@ my-profile:
       priority: interactive
 ```
 
-### Timeouts
+### Timeouts and Retries
+
+<VersionBlock firstVersion="1.1">
+
+Every BigQuery query is made in two steps:
+1. Submit the query job to BQ server and obtain a query job id. 
+2. Wait for the query to execute and poll the query result. 
+
+dbt supports configuring BigQuery query timeouts in each step, as well as the retry and deadline of the overall query. 
+
+#### job_execution_timeout_seconds
+
+The timeout setting of step 2, `job_execution_timeout_seconds`, is the most common one.
```

Comment:
@jtcohen6 do you mean this is the most common configuration to set or to exceed? Or something else?

Reply:
This language all originated from a community member's PR (the person who contributed the feature to `dbt-bigquery`), so I'm not sure exactly what he meant by this. My interpretation is, this is the configuration you're most likely to want to set, as an end user of dbt + BigQuery. I'll update language to reflect